lib/SearchAndSort/SearchAndSort.js

```diff
@@ -1122,7 +1125,7 @@ class SearchAndSort extends React.Component {
           appIcon={<AppIcon app={moduleName} />}
           paneTitle={title || (module && module.displayName)}
           paneSub={paneSub}
-          lastMenu={this.renderNewRecordBtn()}
+          lastMenu={hasNewButton ? this.renderNewRecordBtn() : null}
```

Comment:
As I see there is already the [prop](https://github.com/folio-org/stripes-smart-components/blob/02eb030c260491b1e365e64165349e5ac99f05b8/lib/SearchAndSort/SearchAndSort.js#L789) for this. Please check whether it does what you need.

Reply:
Victor, `disableRecordCreation` prop does not allow to create new record at all, whereas I need to have opportunity to create new record with the `New` button, which is located inside `Actions` dropdown, but not display `New` button by it's own, which is hardcoded so I have creted extra prop.

Reply:
It prevents creation [here](https://github.com/folio-org/stripes-smart-components/blob/02eb030c260491b1e365e64165349e5ac99f05b8/lib/SearchAndSort/SearchAndSort.js#L903)

Reply:
IOW, this is basically a short-term fix to allow "un-migrated" apps to continue displaying the `New` button, and to allow "migrated" apps (those that have added the "new" action to their action menu) to hide it. Correct?

Reply:
Yes, Zak, this is correct. For hiding `New` button, `hasNewButton` prop should be setted to `false`